src/editor.rs

```diff
@@ -11,29 +11,37 @@ pub trait Editor {
     /// Get the coordinates of the editor window.
     fn position(&self) -> (i32, i32);
 
-
     /// Editor idle call. Called by host.
     fn idle(&mut self) {}
 
     /// Called when the editor window is closed.
     fn close(&mut self) {}
 
-    /// Called when the editor window is opened. `window` is a platform dependent window pointer
-    /// (e.g. `HWND` on Windows, `WindowRef` on OSX, `Window` on X11/Linux).
-    fn open(&mut self, window: *mut c_void);
+    /// Called when the editor window is opened.
+    ///
+    /// `parent` is a window pointer that the new window should attach itself to.
+    /// **It is dependent upon the platform you are targeting.**
+    ///
+    /// A few examples:
+    ///
+    ///  - On Windows, it should be interpreted as a `HWND`
+    ///  - On Mac OS X (64 bit), it should be interpreted as a `NSView*`
+    ///  - On X11 platforms, it should be interpreted as a `u32` (the ID number of the parent window)
+    ///
+    /// Return true if the window opened successfully, false otherwise.
+    fn open(&mut self, parent: *mut c_void) -> bool;
 
     /// Return whether the window is currently open.
     fn is_open(&mut self) -> bool;
 
-
     /// Set the knob mode for this editor (if supported by host).
     ///
     /// Return true if the knob mode was set.
```

Comment:
Put `true` in backquotes here as well, while you are at it fixing typos and such. :)